postgrest/base_request_builder.py

```diff
@@ -366,30 +366,45 @@ def __init__(
         BaseFilterRequestBuilder.__init__(self, session, headers, params)
 
     def order(
-        self: _FilterT, column: str, *, desc: bool = False, nullsfirst: bool = False
+        self: _FilterT,
+        column: str,
+        *,
+        desc: bool = False,
+        nullsfirst: bool = False,
+        foreign_table: str = "",
```

Comment:
```suggestion
        foreign_table: Optional[str] = None,
```
I feel like marking the argument as optional and defaulting to None is better than defaulting to an empty string, but that can be debated